SampleScripts/PowerManagement-ManagementDomain.ps1

```diff
@@ -924,30 +923,29 @@ if ($PsBoundParameters.ContainsKey("startup")) {
             else {
                 Set-DrsAutomationLevel -server $vcServer.fqdn -user $vcUser -pass $vcPass -cluster $cluster.name -level $DrsAutomationLevel
             }
+        }
 
-            # Startup the vSphere Cluster Services Virtual Machines in the Management Workload Domain
-            Set-Retreatmode -server $vcServer.fqdn -user $vcUser -pass $vcPass -cluster $cluster.name -mode disable
-            # Waiting for vCLS VMs to be started for ($retries*10) seconds
-            Write-PowerManagementLogMessage -Type INFO -Message "vCLS retreat mode has been set. vCLS virtual machines startup will take some time. Please wait." -Colour Yellow
-            $counter = 0
-            $retries = 10
-            $sleep_time = 30
-            while ($counter -ne $retries) {
-                $powerOnVMcount = (Get-VMsWithPowerStatus -powerstate "poweredon" -server $vcServer.fqdn -user $vcUser -pass $vcPass -pattern "(^vCLS-\w{8}-\w{4}-\w{4}-\w{4}-\w{12})|(^vCLS\s*\(\d+\))|(^vCLS\s*$)" -silence).count
-                if ( $powerOnVMcount -lt 3 ) {
-                    Write-PowerManagementLogMessage -Type INFO -Message "There are $powerOnVMcount vCLS virtual machines running. Sleeping for $sleep_time seconds until the next check."
-                    start-sleep $sleep_time
-                    $counter += 1
-                }
-                else {
-                    Break
-                }
-            }
-            if ($counter -eq $retries) {
-                Write-PowerManagementLogMessage -Type ERROR -Message "The vCLS virtual machines did not start within the expected time. Stopping script execution..." -Colour Red
-                Exit
+        # Startup the vSphere Cluster Services Virtual Machines in the Management Workload Domain
+        Set-Retreatmode -server $vcServer.fqdn -user $vcUser -pass $vcPass -cluster $cluster.name -mode disable
+        # Waiting for vCLS VMs to be started for ($retries*10) seconds
+        Write-PowerManagementLogMessage -Type INFO -Message "vCLS retreat mode has been set. vCLS virtual machines startup will take some time. Please wait."
+        $counter = 0
+        $retries = 10
+        $sleep_time = 30
+        while ($counter -ne $retries) {
+            $powerOnVMcount = (Get-VMsWithPowerStatus -powerstate "poweredon" -server $vcServer.fqdn -user $vcUser -pass $vcPass -pattern "(^vCLS-\w{8}-\w{4}-\w{4}-\w{4}-\w{12})|(^vCLS\s*\(\d+\))|(^vCLS\s*$)" -silence).count
+            if ( $powerOnVMcount -lt 3 ) {
+                Write-PowerManagementLogMessage -Type INFO -Message "There are $powerOnVMcount vCLS virtual machines running. Sleeping for $sleep_time seconds until the next check."
+                start-sleep $sleep_time
```

Comment:
```suggestion
                Start-Sleep $sleep_time
```

Reply:
addressed in commit a0e6a231b99ac37596cd1b07cf674e22d71bda11